systems/plants/URDFRigidBodyManipulator.cpp

```diff
@@ -345,6 +339,31 @@ bool URDFRigidBodyManipulator::addURDF(boost::shared_ptr<urdf::ModelInterface> _
       dofnum[index-1] = _dofnum;
     }
 
+    if(l->second->inertial != nullptr)
+    {
+      boost::shared_ptr<urdf::Inertial> inertial = l->second->inertial;
+      bodies[index]->mass = inertial->mass;
+      bodies[index]->com << inertial->origin.position.x, inertial->origin.position.y, inertial->origin.position.z, 1.0;
+      Matrix3d I3d;
+      I3d << inertial->ixx, inertial->ixy, inertial->ixz,
+             inertial->ixy, inertial->iyy, inertial->iyz,
+             inertial->ixz, inertial->iyz, inertial->izz;
+      Vector4d quat;
+      quat << inertial->origin.rotation.w, inertial->origin.rotation.x, inertial->origin.rotation.y, inertial->origin.rotation.z;
+      auto R = quat2rotmat(quat);
+      I3d = R*I3d*R.transpose();
+
+      // from mcI.m
+      Matrix3d C;
+      C <<  0,                     -bodies[index]->com(2),   bodies[index]->com(1),
+            bodies[index]->com(2), 0,                       -bodies[index]->com(0),
+           -bodies[index]->com(1), bodies[index]->com(0),   0 ;
+      bodies[index]->I.block(0,0,3,3) << I3d + bodies[index]->mass*C*C.transpose();
+      bodies[index]->I.block(0,3,3,3) << bodies[index]->mass*C;
+      bodies[index]->I.block(3,0,3,3) << bodies[index]->mass*C.transpose();
+      bodies[index]->I.block(3,3,3,3) << bodies[index]->mass*Matrix3d::Identity();
+    }
+
```

Comment:
There's a method called transformSpatialInertia in drakeGeometryUtil taht does all this.